CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
Yes, I'm using the latest master, and I confirm you that setting ```progressRingTintColor``` it works as shown in your screenshot.

But if you instead set only ```tintColor``` you will notice that the Today button will stay red instead of purple, and only the selected day background goes purple.